### DISCUSSION TITLE
Netbox worker user is hammering ldap

### DISCUSSION THREAD

**Discussion by deligatedgeek** [netbox-community/netbox#12230] [Help Wanted!]
### NetBox version

v3.4.5

### Python version

3.9

### Steps to Reproduce

Use many tasks on workers with LDAP authentication generates several hundred authentication requests per second

### Expected Behavior

Some kind of LDAP user caching, or configuration that reduces LDAP load

### Observed Behavior

up to 63,000 LDAP authentication requests per hour

**Comment by kkthxbye-code** [comment]
This is in no way detailed enough to warrent a bug report so I'll convert it to a discussion.

As you gave barely any information, I can only guess as to the issue, but usually if API requests are triggering the LDAP traffic, the solution is to use group mirroring and disabling AUTH_LDAP_FIND_GROUP_PERMS. The ldap cache does not function with the API sadly.

```
AUTH_LDAP_FIND_GROUP_PERMS = False
AUTH_LDAP_MIRROR_GROUPS = True # Or a list of groups you want to mirror.
```

More documentation here: https://django-auth-ldap.readthedocs.io/en/latest/

**Reply by deligatedgeek** [reply]
Thanks for the quick response, this was setup by a 3rd party before they went on holiday.
There are a lot of integrations and moving parts.
Is there a better/preferred way to configure API authentication?
